### DISCUSSION TITLE
拉个群吧

### DISCUSSION THREAD

**Discussion by Attiv** [imfuxiao/Hamster#64] [Show and tell]
我们天天在 irime 群里讨论仓，多少有点不礼貌吧。。

**Comment by imfuxiao** [comment]
您好, 群的问题, 我也一直在想, 国内的群有zz责任, 需要群主负责, 国外软件群需要科学上网才行. 有点纠结

**Reply by Attiv** [reply]
那这不就点名 slack 了么。。不在国内，还不用科学

**Reply by imfuxiao** [reply]
咦~slack不用fq么?

**Reply by Attiv** [reply]
应该是不用的，可能偶尔用吧，最新没开也能用

**Reply by wzxmer** [reply]
> 应该是不用的，可能偶尔用吧，最新没开也能用

不用，我最近玩了下claude，不需要FQ就能用

**Comment by imfuxiao** [comment]


**Reply by dnxxymcj** [reply]
大佬们解释一下，真不懂
